WalletWasabi.Documentation/HowToDeploy.md

```diff
@@ -23,15 +23,24 @@
 
    Note: As you can see, the deployer can choose the `codename` freely.
 
-2. Deploy using the `deploy.sh` script.
+   - Creating a tag: `git tag v2.0.3-myexampletag`
+   - Push the tag to upstream: `git push upstream v2.0.3-myexampletag`
+
+3. Deploy using the `deploy.sh` script.
 
 # Templates for communication
 
-**Deploy to TestNet example**
+**Deploy example**
+
+Post the following message to slack/integrations channel
 
-Hello there, we are deploying this commit to the `TestNet` server.
+```
+Hello there, we are deploying this commit to backend servers.
 - Latest commit on backend: 167c81be80d8d3de9deaf8d306017c5403593c89
 - Planning to deploy to backend: 460e21ce71738d3cc1560a3d4fc1984cc4beb725
 - PRs with affiliate label: https://github.com/zkSNACKs/WalletWasabi/pulls?q=is%3Apr+is%3Aclosed+label%3Aaffiliate
 
 Please ack and test.
+
+Tagging: @Kuba Ondracek
```

Comment:
I wouldn't add personal info here. Instead, I would automate all this thing. Deploying should be done 100% automatic.

Reply:
Guys this is not productive at all. I am not refactoring the whole procedure here, just apply new information and  adjust to the current flow. Do not push things 10 steps forward. Automation can come if we set the procedure in stone. Which means it have not changed in the last 6 months and we have the time and resource - neither we have.

Reply:
> I wouldn't add personal info here.

So how do we remember then?

Reply:
The tool can remind it to you. See: https://github.com/zkSNACKs/BackendScripts/pull/18

Reply:
The goal is to automate things to make them easier. The `deploy.sh` script made the deployment task much easier and safer than the documented procedures and I don't want to go back to those times.